staker-rewards/README.adoc

```diff
@@ -102,7 +102,7 @@ to a beneficiary.
 To calculate operator rewards and calculate merkle tree, drop option `-v`, and run:
 
```

Comment:
Can we add a section on how to run `block-by-date.js` to obtain start block and end block? Ideally, we'd also add `.sh` script for convenience.